### PR TITLE
Editorial: Add mention to RelativeTimeFormat to Api Overview

### PR DESCRIPTION
**Pull request by romulocintra**
Ref #468 

Add mention to RelativeTimeFormat to Api Overview

### CODE REVIEW

File: spec/overview.html [hunk 1]
```diff
@@ -23,7 +23,7 @@ <h1>API Overview</h1>
...
...      The ECMAScript 2022 Internationalization API Specification provides several key pieces of language-sensitive functionality that are required in most applications: String comparison (collation), number formatting, date and time formatting, relative time formatting, display names, list formatting, pluralization rules, and case conversion. While the ECMAScript 2022 Language Specification provides functions for this basic functionality (on <emu-xref href="#sec-array.prototype">Array.prototype</emu-xref>: `toLocaleString`; on <emu-xref href="#sec-string.prototype">String.prototype</emu-xref>: `localeCompare`, `toLocaleLowerCase`, `toLocaleUpperCase`; on <emu-xref href="#sec-number.prototype">Number.prototype</emu-xref>: `toLocaleString`; on <emu-xref href="#sec-date.prototype">Date.prototype</emu-xref>: `toLocaleString`, `toLocaleDateString`, and `toLocaleTimeString`), it leaves the actual behaviour of these functions largely up to implementations to define. The ECMAScript 2022 Internationalization API Specification provides additional functionality, control over the language and over details of the behaviour to be used, and a more complete specification of required functionality.
```
Comment (romulocintra) [inline spec/overview.html:26]: We may also mention Intl.Locale by adding something like "locale selection and operation" together with 

> ...String comparison (collation), number formatting, date and time formatting, relative time formatting, display names, list formatting, pluralization rules ...

Comment (ryzokuken) [reply spec/overview.html:26]: Sounds good!

File: spec/overview.html [hunk 2]
```diff
@@ -32,7 +32,7 @@ <h1>API Overview</h1>
...
       </li>
       <li>
-        Indirectly, by using the functions of the ECMAScript 2022 Language Specification mentioned above. The collation and formatting functions are respecified in this specification to accept the same arguments as the Collator, NumberFormat, and DateTimeFormat constructors and produce the same results as their compare or format methods. The case conversion functions are respecified to accept a list of preferred languages.
+        Indirectly, by using the functions of the ECMAScript 2022 Language Specification mentioned above. The collation and formatting functions are respecified in this specification to accept the same arguments as the Collator, NumberFormat, RelativeTimeFormat and DateTimeFormat constructors and produce the same results as their compare or format methods. The case conversion functions are respecified to accept a list of preferred languages.
```
Comment (ryzokuken) [inline spec/overview.html:35]: Is `RelativeTimeFormat#format` called in any ecma262 functions? I couldn't recall any but just wanted to ask.

**Review by ryzokuken — COMMENTED** [review]
Hello! Thanks for the PR, please take a look to the comments below.

File: spec/overview.html (hunk 2 above)
Comment (romulocintra) [reply spec/overview.html:35]: Was added based on https://github.com/tc39/ecma402/issues/468#issue-644449434

File: spec/overview.html (hunk 1 above)
Comment (romulocintra) [reply spec/overview.html]: Just updated with `locale selection and operation`

File: spec/overview.html (hunk 2 above)
Comment (gibson042) [reply spec/overview.html:35]: This section defines how applications indirectly use ECMA-402 through ECMA-262 overrides (described at https://tc39.es/ecma402/#locale-sensitive-functions , none of which reference part of RelativeTimeFormat).
```suggestion
        Indirectly, by using the functions of the ECMAScript 2022 Language Specification mentioned above. The collation and formatting functions are respecified in this specification to accept the same arguments as the Collator, NumberFormat, and DateTimeFormat constructors and produce the same results as their compare or format methods. The case conversion functions are respecified to accept a list of preferred languages.
```

**Review by gibson042 — CHANGES_REQUESTED** [review]

**Review by gibson042 — APPROVED** [review]

**Review by ryzokuken — APPROVED** [review]

**Comment by ryzokuken** [comment]
Thanks!

**Pull request merged**